storage/namespace/config.go

```diff
@@ -34,13 +34,13 @@ type MapConfiguration struct {
 
 // MetadataConfiguration is the configuration for a single namespace
 type MetadataConfiguration struct {
-	ID                  string                  `yaml:"id" validate:"nonzero"`
-	NeedsBootstrap      *bool                   `yaml:"needsBootstrap"`
-	NeedsFlush          *bool                   `yaml:"needsFlush"`
-	WritesToCommitLog   *bool                   `yaml:"writesToCommitLog"`
-	NeedsFilesetCleanup *bool                   `yaml:"needsFilesetCleanup"`
-	NeedsRepair         *bool                   `yaml:"needsRepair"`
-	Retention           retention.Configuration `yaml:"retention" validate:"nonzero"`
+	ID                string                  `yaml:"id" validate:"nonzero"`
+	NeedsBootstrap    *bool                   `yaml:"needsBootstrap"`
```

Comment:
could you rename all these to the new `...Enabled` convention too

Reply:
done